### PR TITLE
.github: print date of today in the PR title

### PR DESCRIPTION
**Pull request by dongsupark**
Let's add date of today in the PR title, to avoid confusion when seeing the same title for multiple different weekly update PRs.

### CODE REVIEW

**Comment by dongsupark** [comment]
Tested via my fork. https://github.com/dongsupark/portage-stable/pull/11
Works well.

File: .github/workflows/update-packages-from-list.yml [hunk 1]
```diff
@@ -73,6 +75,6 @@ jobs:
           branch: buildbot/automatic-package-updates
```
Comment (krnowak) [inline .github/workflows/update-packages-from-list.yml:75]: Question that I have here is what would be the desired outcome when the weekly package updates PR is not handled within the week:
- create another PR next Monday (would need a differently named branch too)
- update the existing PR (stays as is)

What do you think?

Comment (dongsupark) [reply .github/workflows/update-packages-from-list.yml:75]: I would prefer to have another PR for a new week.
If we would explicitly want to merge new updates into an existing PR, then we could simply close the existing one, and run new Github Actions.

Comment (dongsupark) [reply .github/workflows/update-packages-from-list.yml]: Done

**Review by krnowak — APPROVED** [review]

**Pull request merged**